src/XrdPfc/XrdPfcIOFile.cc

```diff
@@ -83,7 +83,7 @@ int IOFile::initCachedStat(const char* path)
    int res = -1;
    struct stat tmpStat;
 
-   if (m_cache.GetOss()->Stat(path, &tmpStat) == XrdOssOK)
+   if (m_cache.GetOss()->Stat(GetFilename().c_str(), &tmpStat) == XrdOssOK)
```

Comment:
We don't have the time of the last write to data-file -- it could be added to the cinfo file. Technically one could divine it from the access records in cinfo but those get merged at some point and become less and less precise time-wise.

Reply:
> We don't have the time of the last write to data-file -- it could be added to the cinfo file

Nah, I don't think this is necessary.  I'm happy using what the OS itself is reporting as the last write to the file.